### ISSUE TITLE
Mac: MediaPlayer.Control.IsPlaying returns true when the GameObject is disabled (and video is thus paused)

### ISSUE THREAD

**Issue by TheloTheGreat** [RenderHeads/UnityPlugin-AVProVideo#652]
On Mac, open the Demo_MediaPlayer scene, add this to MediaPlayerUI.Update:
```
if (Input.GetKeyDown(KeyCode.A)) {
	Debug.Log("IsPlaying: " + _mediaPlayer.Control.IsPlaying());
}
```

Launch the scene, then while the demo video is playing, manually deactivate the MediaPlayer GameObject or disable its MediaPlayer component. The video then auto-pauses. Then press A.
Expected result: "IsPlaying: False"
Actual result: "IsPlaying: True"

Note: On Windows, IsPlaying does correctly return false while the GameObject is deactivated or its MediaPlayer component is disabled.

**Comment by MorrisRH** [comment]
A fix will be available in the next release.

**Comment by TheloTheGreat** [comment]
Fix confirmed in 2.0.9, thanks.
